docs/installing_deis/aws.rst

```diff
@@ -102,11 +102,24 @@ in `cloudformation.json`_. For example, to configure all of the optional setting
     {
         "ParameterKey":     "ELBScheme",
         "ParameterValue":   "internal"
+    },
+    {
+        "ParameterKey":     "RootVolumeSize",
+        "ParameterValue":   "100"
```

Comment:
When someone comes to the docs, they may be confused on what unit of measurement this is. Maybe we should add a note stating that this is in gigabytes?